src/mbgl/text/collision_index.cpp

```diff
@@ -122,13 +122,13 @@ IntersectStatus CollisionIndex::intersectsTileEdges(const CollisionBox& box,
     const float tileY2 = tileEdges[3];
 
     // Check left border
-    int minSectionLength = std::min(tileX1 - x1, x2 - tileX1);
+    float minSectionLength = std::min(tileX1 - x1, x2 - tileX1);
```

Comment:
Seems to break some render tests...

Reply:
Why this happens is obscure to me. If you look closer there is a cast to `int` when this variable is copied over to the `result`.

Reply:
Also it would be useful to actually to paste what tests this breaks to see the effect.